MaxText/configs/base.yml

```diff
@@ -206,6 +206,10 @@ load_from_prefill_dir: False # If true, decode.py doesn't "prefill" but just rea
 prefill_cache_dir: "" # If set and load_from_prefill_dir, decode.py reads from directory. If set, decode.py writes to directory
 autoregressive_decode_assert: ""
 
+# For nsys profiler, pass the training command to nsys command
+# e.g. nsys profile -s none --force-overwrite true --capture-range=cudaProfilerApi --capture-range-end=stop {training command} 
+profiler: "" # Supported profiler: '', xprof, nsys
+# If set to true and profiler is empty, xprof is selected as a profiler
 enable_profiler: False
 # If set to true, upload all profiler xplane results from all hosts. Otherwise, only upload the xplane result from the first host.
```

Comment:
nit: Remove xplane from the comments given we support both xprof and nsys now?

`If set to true, upload all profiler results from all hosts. Otherwise, only upload the profiler result from the first host.`